VirtusLab-Open-Source/strapi-plugin-navigation
# feat: prefering custom content type in settings page

## Ticket

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/issues/430

## Summary

![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/13495487/49eee2e3-5b7c-4edd-9f9b-60fa968d989f)

Added option to prefer only custom content types in content type pickers

## Test Plan

- start server
- open navigation settings page
- enable picking only custom content types
- only custom content types are present
- disable picking only custom content types
- pick `File` content type
- enable picking custom content types
- content types select should preserve already picked content types
